config.json.example

```diff
@@ -0,0 +1,14 @@
+{
+  "DEBUG": false,
+
+  "BACKEND_URL": "http://192.168.1.32/shoov.local/",
+
+  "LOGGLY_TOKEN": "704620e7-c3d4-4d84-a0c1-926e5147592c",
```

Comment:
this shouldn't hold any Gizra related info. just replace with `<YOUR-LOGGLY-TOKEN>`, `<YOUR-LOGGLY-SUBDOMAIN>`, etc'